contracts/utils/deploy.js

```diff
@@ -1248,6 +1248,16 @@ function deploymentWithGuardianGovernor(opts, fn) {
   return main;
 }
 
+async function replaceContractAt(targetAddress, mockContract) {
```

Comment:
On my branch, I moved `replaceContractAt` to a new `utils/hardhat.js` file with `hardhatSetBalance`.

https://github.com/OriginProtocol/origin-dollar/pull/1846/files#diff-46bda6023853d155885af601cb43da6cab0ccea9b57fd3bb9b14c89ebcbdf8a6